src/blog/2019-02-13/announcing-stryker-1-0.pug

```diff
@@ -0,0 +1,113 @@
+extends ../../layout/blog-layout.pug
+block content
+    :markdown-it
+    
+        If you’re unfamiliar with Stryker and mutation testing, now is the perfect time to try it out! 
+        We've updated [our quickstart](/stryker/quickstart), so go ahead and start using Stryker today.
+        Stryker can help you to improve your test quality by inserting bugs in your code and validating
+        that your tests can find them. The output is a comprehensive report telling you exactly what your unit tests missed.
+
+        ## What's new
+
+        Here you find a list of all the new features in Stryker 1.0 (alphabetical order):
+
+    figure
+        img(src="/images/blogs/tumble-weed.jpg")
+        figcaption 
+            em Nothing to see here
+
+    :markdown-it
+        That's right! We chose to not introduce new features. Instead, we've _removed all deprecated functionality_.
```

Comment:
Maybe you could add how much deprecated functionality was removed, (but reassuring that no non-deprecated functionality was broken).

Reply:
In my opinion this is already covered by running stryker against the latest 0.x version. We have warning messages with suggestions on fixing the deprecated functionality specifically in your config.